tests/main.js

```diff
@@ -8,17 +8,17 @@ const formattedText = `
 !!!  Unicorns are lit !!!
 Hello this is a formatted text !
 				It has alignements
-				already includes 
-				in it. 
+				already includes${' '}
+				in it.${' '}
 Boxen should protect this alignement,
 		otherwise the users would be sad !
-Hehe          Haha                                 
+Hehe          Haha${' '.repeat(33)}
```

Comment:
I don't know if the space is useful in tests, but https://github.com/sindresorhus/boxen/blob/badc4bc7e89074ab6cc111c99088c6f6fc15fdd9/.editorconfig#L7, we should not write it directly.